src/databricks/labs/ucx/source_code/known.py

```diff
@@ -63,7 +63,7 @@ def as_dict(self):
 _DEFAULT_ENCODING = sys.getdefaultencoding()
 
 
-class Whitelist:
+class AllowList:
```

Comment:
can you rename this class to `Known`? allow_list is fine for varnames.

Reply:
Yeah that's a better name, thanks. Done.